spec/rspec/matchers/built_in/yield_spec.rb

```diff
@@ -563,6 +563,30 @@ def each_arg(*args, &block)
     expect(val).to be_nil
   end
 
+  it "works correctly when the method yield multiple args each time (passing case)" do
```

Comment:
But your comment helped me realize "yield" should be "yields" here so I fixed that :).